### ISSUE TITLE
The reward on the validation set of the CALX example has been staying around 0.5 and is unable to improve or increase.

### ISSUE THREAD

**Issue by johnson7788** [microsoft/agent-lightning#454]
Why does the reward on the validation set of my CALX example stay around 0.5 and fail to improve? I am using a 0.5B-parameter model—could the model be too small, or is the reward too sparse? How should I improve it?

<img width="2792" height="1340" alt="Image" src="https://github.com/user-attachments/assets/d1fa0fcc-ef5d-427e-8a6a-f74564455a50" />

**Comment by ultmaster** [comment]
I think the performance this saturating under this configuration. You might want to tune some hyper-parameters if you want to make it better. But I doubt whether it has some real use cases since the model size is so small.
